bin/lib/log-to-spoor.js

```diff
@@ -0,0 +1,38 @@
+const debug = require('debug')('bin:middleware:log-requests');
+const fetch = require('node-fetch');
+
+module.exports = function(data){
+
+	if(!data){
+		return Promise.resolve({'status' : 'Failed. Invalid data passed.'});
+	}
+
+	debug('Sending data to spoor', data);
+
+	return fetch('https://spoor-api.ft.com/ingest', {
+			method: 'POST',
+			headers: {
+				'Accept': 'application/json',
+				'Content-Type': 'application/json',
+				'Content-Length': new Buffer(JSON.stringify(data)).length,
+				'User-Agent': 'ft-labs/v1.0'
```

Comment:
I would add a loud comment here highlighting how important it is to specify the User-Agent